MatrixSDK/Data/MXRoom.m

```diff
@@ -2380,6 +2389,40 @@ - (MXHTTPOperation *)sendPollEndForEvent:(MXEvent *)pollStartEvent
     return [self sendEventOfType:[MXTools eventTypeString:MXEventTypePollEnd] content:content localEcho:localEcho success:success failure:failure];
 }
 
+#pragma mark - Location sharing
+
+- (MXHTTPOperation *)sendLocationWithLatitude:(double)latitude
+                                    longitude:(double)longitude
+                                  description:(NSString *)description
+                                    localEcho:(MXEvent **)localEcho
+                                      success:(void (^)(NSString *))success
+                                      failure:(void (^)(NSError *))failure
+{
+    NSMutableDictionary *content = [NSMutableDictionary dictionary];
+    content[kMXMessageTypeKey] = kMXMessageTypeLocation;
+    
+    NSDictionary *locationContent = [[MXEventContentLocation alloc] initWithLatitude:latitude
+                                                                           longitude:longitude
+                                                                         description:description].JSONDictionary;
+    
+    content[kMXMessageContentKeyExtensibleLocationMSC3488] = locationContent;
+    
+    NSString *geoURI = locationContent[kMXMessageContentKeyExtensibleLocationURI];
```

Comment:
Does it make sense to create a `geoURI` property or method on `MXEventContentLocation` instead?

Reply:
Yep, good idea 👍